src/test/java/teammates/test/cases/action/AdminEmailComposeSaveActionTest.java

```diff
@@ -52,7 +52,9 @@ public void testExecuteAndPostProcess() throws Exception {
                         Const.ParamsNames.ADMIN_EMAIL_ADDRESS_RECEIVERS, receiver);
         ShowPageResult pageResult = getShowPageResult(action);
         assertEquals(
-                Const.ViewURIs.ADMIN_EMAIL + "?error=false&user=admin.user", pageResult.getDestinationWithParams());
+                getPageResultDestination(
+                        Const.ViewURIs.ADMIN_EMAIL, false, adminUserId),
+                pageResult.getDestinationWithParams());
```

Comment:
Sometimes parameter of getPageResultDestination exceeds 120 chars so have to break in multiple lines. But here and few other places it can be moved to one Line. Would update that.